zkevm-circuits/src/super_circuit.rs

```diff
@@ -272,7 +263,10 @@ impl<const MAX_TXS: usize, const MAX_CALLDATA: usize> SuperCircuit<Fr, MAX_TXS,
     ///
     /// Also, return with it the minimum required SRS degree for the circuit and
     /// the Public Inputs needed.
-    pub fn build(geth_data: GethData, rng: impl RngCore + Clone) -> (u32, Self, Vec<Vec<Fr>>) {
+    pub fn build(
+        geth_data: GethData,
+        rng: impl RngCore + Clone,
```

Comment:
Why not getting &mut impl ... as usual?

Reply:
No reason in particular :P
Updated in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/706/commits/732db2e3a74100ea38f729b6ea32da8555b00a41